src/Splunk.OpenTelemetry.AutoInstrumentation/Logging/Logger.cs

```diff
@@ -46,7 +46,7 @@ static Logger()
         }
     }
 
-    internal static void Warning(string message)
+    void ILogger.Warning(string message)
```

Comment:
Can we also create a unit test for this class to make sure that there is no expectation when using reflection?

It would be a safety net if the internal logging API in the upstream would change.

Reply:
Ahh... you are right. We catch the exception... I would like to have some test to make sure that we are properly calling the upstream API via reflection 😬 Maybe we should extract only the reflection code and have a unit test for it?